client/src/components/LoginButton.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
change this component to loader and move this in a separate folder
/components/loaders/Loader.jsx something like that for structurize format

Reply:
done